test/e2e/azure_test.go

```diff
@@ -155,6 +156,7 @@ var _ = Describe("Workload cluster creation", func() {
 				clusterName = getClusterName(clusterNamePrefix, "public-custom-vnet")
 				By("Creating a custom virtual network", func() {
 					Expect(os.Setenv(AzureCustomVNetName, "custom-vnet")).To(Succeed())
+					Expect(os.Setenv(AzureCustomVnetResourceGroup, clusterName+"-vnetrg")).To(Succeed())
```

Comment:
is everything getting cleaned up as it should now that we're using a separate RG?

Reply:
Not sure, but I'll check

Reply:
Yep, I don't see any extra resource groups around after running the test locally.